content/api/commands/clearalllocalstorage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: clearAllLocalStorage
+---
+
+Clear localStorage data for all origins with which the test has interacted.
+
+<Alert type="warning">
+
+Cypress automatically runs this command _before_ each test to prevent state from
```

Comment:
```suggestion
Cypress automatically runs this command _before_ each test, when [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation) is `on`, to prevent state from
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4876/commits/1b31d3d09ba10bea4afcd9cdc34af0fd7480bd79